scripts/globals/weaponskills.lua

```diff
@@ -248,9 +248,19 @@ local function getMultiAttacks(attacker, target, wsParams)
         end
     end
 
-    numHits = utils.clamp(numHits + bonusHits, 1, 8)
```

Comment:
I should have been more clear in the original post, but I have a screenshot showing a possible 9 hit. Dancing edge while dual welding and proc multi hit in each initial

Reply:
> [10:49 AM]MowFord: it's not clamping the total hits
> [10:49 AM]MowFord: it's clamping hitsDone
> [10:49 AM]MowFord: which isn't accounting for the offhand hit

Reply:
Just to play devil's advocate I guess...

Are we ABSOLUTELY sure extra hit(s)/swing(s) perceived past the WS limit were not a seperate trigger outside of the WS (ie, retaliation, unfortunate timing of a nornal attack round proc, etc.)?

Reply:
Yes because I'm not confirming via tp gains or anything. In my OP you can see I'm confirming via a print in the `weaponskills.lua` function, just before the return